Platform/UTMJailbreak.c

```diff
@@ -41,17 +45,26 @@ kern_return_t catch_exception_raise(mach_port_t exception_port,
     return KERN_FAILURE;
 }
 
+static void init_sys_functions(void) {
+    static bool init_done = false;
+    if (!init_done) {
+        csops = dlsym(dlopen(NULL, RTLD_LAZY), "csops");
+        exc_server = dlsym(dlopen(NULL, RTLD_LAZY), "exc_server");
+        ptrace = dlsym(dlopen(NULL, RTLD_LAZY), "ptrace");
```

Comment:
Have you considered weak-linking these and letting the dynamic linker do this work for you?

Reply:
Well I got the code from you: https://github.com/hrydgard/ppsspp/commit/99c631c79113969411c6ce24a4ab80f1d89b7369

Why even weak link? Previously I just had them linked and it works fine. I thought you had some more subtle reason.

Reply:
Huh, so you did. I seem to recall using `dlsym` so I could name my function pointers the same as library functions, but I don't know why I didn't just declare a prototype for them… If you can just link directly, I would say go for it and not to even bother with weak linking. If Apple removes those symbols from the list of things they'll allow you to link against, we can dive more into making weak linking work.